core/string/print_string.h

```diff
@@ -51,6 +51,8 @@ void remove_print_handler(PrintHandlerList *p_handler);
 
 extern bool _print_line_enabled;
 extern bool _print_error_enabled;
+extern bool _stdout_enabled;
+extern bool _stderr_enabled;
```

Comment:
I'm not so fond of adding more globals in a file which is included virtually everywhere. But there's a precedent with the `_print_line_enabled` and `_print_error_enabled` so I understand it makes sense to just keep the same structure.

Might need a refactor further down the line.

Reply:
Yeah, I see that. Maybe we can add this to OS?